src/patches.js

```diff
@@ -1,100 +1,72 @@
 import {each} from "./common"
 
-export function generatePatches(
-    state,
-    basepath,
-    patches,
-    inversePatches,
-    baseValue,
-    resultValue
-) {
-    if (patches)
-        if (Array.isArray(baseValue))
-            generateArrayPatches(
-                state,
-                basepath,
-                patches,
-                inversePatches,
-                baseValue,
-                resultValue
-            )
-        else
-            generateObjectPatches(
-                state,
-                basepath,
-                patches,
-                inversePatches,
-                baseValue,
-                resultValue
-            )
+export function generatePatches(...args) {
```

Comment:
Oh, this is probably inflated by the transpiler. Should I revert this change?

Reply:
Yeah I think minifier will make this pretty small anyway